cmd/main.go

```diff
@@ -155,13 +154,14 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&corecontrollers.QueueReconciler{
-		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
-	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "Queue")
-		os.Exit(1)
-	}
+	//if err = (&corecontrollers.QueueReconciler{
```

Comment:
Commenting this out for now to avoid any NPEs in this intermediate state where the queue client can be nil.